### PR TITLE
chore: sort categories in recommended app service response

### PR DESCRIPTION
**Pull request by BenjaminX**

### CODE REVIEW

**Review by Yeuoly — APPROVED** [review]
LGTM

**Comment by loganclark360** [comment]
👍

W. Logan Clark

File: api/services/recommended_app_service.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def _fetch_recommended_apps_from_db(cls, language: str) -> dict:
 
             categories.add(recommended_app.category)  # add category to categories
 
-        return {'recommended_apps': recommended_apps_result, 'categories': list(categories)}
+        return {'recommended_apps': recommended_apps_result, 'categories': sorted(list(categories))}
```
Comment (bowenliang123) [inline api/services/recommended_app_service.py:98]:
```suggestion
        return {'recommended_apps': recommended_apps_result, 'categories': sorted(categories)}
```
Simply sort the set, and the returned value is in list type. eg.
image

Comment (BenjaminX) [reply api/services/recommended_app_service.py:98]: list(categories) duplicate removal, and sorted by it.

Comment (bowenliang123) [reply api/services/recommended_app_service.py:98]: Thanks. SGTM.

**Pull request merged**